components/script/dom/clipboard.rs

```diff
@@ -40,6 +90,89 @@ impl Clipboard {
 }
 
 impl ClipboardMethods<crate::DomTypeHolder> for Clipboard {
+    /// <https://w3c.github.io/clipboard-apis/#dom-clipboard-readtext>
+    fn ReadText(&self, can_gc: CanGc) -> Rc<Promise> {
+        // Step 1 Let realm be this's relevant realm.
```

Comment:
You can do `let global =` for this, because that is what matches what the spec means with the relevant realm(at least how we use it to create promises).